.github/workflows/gen-client.yml

```diff
@@ -16,6 +16,15 @@ jobs:
         run: |
           curl -H "Authorization: token ${{ secrets.GH_CQ_BOT }}" https://raw.githubusercontent.com/cloudquery/cloud/main/internal/servergen/spec.json -o spec.json
 
+      - name: Format Specs File
+        run: |
+          docker run --rm -v $(pwd):/local openapitools/openapi-generator-cli generate -i /local/spec.json -g openapi --skip-validate-spec -o /local/generated
```

Comment:
Can we pin the image version so we don't get any surprises when they do breaking changes?

Reply:
done